packages/terra-action-header/src/ActionHeaderContainer.scss

```diff
@@ -0,0 +1,60 @@
+:local {
+  .flex-header {
+    background-color: var(--terra-action-header-background-color, #f4f4f4);
+    border-bottom: var(--terra-action-header-border-bottom, 1px solid #dedfe0);
+    border-left: var(--terra-action-header-border-left);
+    border-right: var(--terra-action-header-border-right);
+    border-top: var(--terra-action-header-border-top);
+    box-shadow: var(--terra-action-header-box-shadow);
+    display: flex;
+    justify-content: space-between;
+    padding: var(--terra-action-header-padding, 0.714rem);
+    position: relative;
+    width: 100%;
+  }
+
+  .flex-fill {
+    align-self: center;
+    flex: 1 1 auto;
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
+
+  .flex-end {
+    align-items: center;
+    display: flex;
+    flex: 0 0 auto;
+    height: 100%;
+    position: relative;
+  }
+
+  .flex-collapse {
+    align-items: center;
+    display: flex;
+    flex: 0 0 auto;
+    height: 100%;
+    max-width: 40%;
+    position: relative;
+  }
+
+  .title-container {
+    height: 100%;
+    left: 0;
```

Comment:
Left and Top values can be removed since it's no longer using absolute positioning.